prototypes/prototype.adoc

```diff
@@ -0,0 +1,44 @@
+---
+authors: Han Solo <han@example.com>, Frodo Baggins <frodo@example.com>
+state: predraft
+---
+= RFD <NUMBER> <Title>
+:author: Han Solo <han@example.com>; Frodo Baggins <frodo@example.com>
```

Comment:
I updated `rfdlint` to validate the titles for `.adoc` files.  `rfdlint` is using `=` as its doc title token, not `#`:  https://github.com/joyent/rfd/blob/master/tools/rfdlint#L131-L133

As for the chat bots, I haven't touched them but suspect it would be a similar change.